### PR TITLE
Content Template

### PR DESCRIPTION
**Pull request by ctf-umbraco**

### CODE REVIEW

File: Umbraco-Uno/Creating-Content/How-To-Setup-Content-Template/index.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+Umbraco Uno offers the option to make a template of your page, to get a nice work process, so that you won't have to set the same page up multiple times.
+
+## What is a Content Template
+
+A Content Template, lets you create a page containing a set up of widgets.
+This is a very useful feature as it allows you to have some predefined templates to choose from.
+
+## How to set it up
+
+Setting up your template comes in two parts, first you need to make a page that has the widgets you want, and then turn that into a template.
```
Comment (Jay-umbr) [inline Umbraco-Uno/Creating-Content/How-To-Setup-Content-Template/index.md:16]: Setting up your template comes in two parts - first you need to make a page that has the widgets you want, and then turn that into a template.

File: Umbraco-Uno/Creating-Content/How-To-Setup-Content-Template/index.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+### Setting up the widgets of the page
+
+Let's start by setting up the page with some widgets, you can read more about [Widgets](../../Uno-pedia/Widgets/index.md) in the Uno-pedia section.
```
Comment (Jay-umbr) [inline Umbraco-Uno/Creating-Content/How-To-Setup-Content-Template/index.md:20]: Start by setting up the page with some widgets (you can read more about [Widgets](../../Uno-pedia/Widgets/index.md) in the Uno-pedia section).

File: Umbraco-Uno/Creating-Content/How-To-Setup-Content-Template/index.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+1. Click the  button in the three ... at your home node and select page
```
Comment (Jay-umbr) [inline Umbraco-Uno/Creating-Content/How-To-Setup-Content-Template/index.md:22]: 1. Click the context menu on your home node (three dots symbol) and select the page

File: Umbraco-Uno/Creating-Content/How-To-Setup-Content-Template/index.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+2. Name the page
+3. Add the Widgets you want in your template and set it up how you like it
+
+:::note It is recommended to not fill out the widgets, because if you do, then you will have to delete the content you don't want each time you use the template.:::
+
+### Making the Content Template
+
+Now that we have a page that we want to use for our template.
+
+1. Click the "..." button in the three at the page, you created in the steps above
```
Comment (Jay-umbr) [inline Umbraco-Uno/Creating-Content/How-To-Setup-Content-Template/index.md:32]: 1. Click the context menu ( ... ) in your content tree on the page you created in the steps above

File: Umbraco-Uno/Creating-Content/How-To-Setup-Content-Template/index.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+versionFrom: 8.0.0
+---
+
+# Content Template
+
+Umbraco Uno offers the option to make a template of your page, to get a nice work process, so that you won't have to set the same page up multiple times.
+
+## What is a Content Template
+
+A Content Template, lets you create a page containing a set up of widgets.
```
Comment (sofietoft) [inline Umbraco-Uno/Creating-Content/How-To-Setup-Content-Template/index.md:11]:
```suggestion
A Content Template lets you create a page containing a complete set of widgets.
```

File: Umbraco-Uno/Creating-Content/How-To-Setup-Content-Template/index.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+This is a very useful feature as it allows you to have some predefined templates to choose from.
+
+## How to set it up
+
+Setting up your template comes in two parts - first you need to make a page that has the widgets you want, and then turn that into a template.
+
+### Setting up the widgets of the page
+
+Start by setting up the page with some widgets.
+You can read more about [Widgets](../../Uno-pedia/Widgets/index.md) in the Uno-pedia section.
```
Comment (sofietoft) [inline Umbraco-Uno/Creating-Content/How-To-Setup-Content-Template/index.md:21]:
```suggestion
You can read more about [Widgets](../../Uno-pedia/Widgets/index.md) in the Uno-pedia.
```

File: Umbraco-Uno/Creating-Content/How-To-Setup-Content-Template/index.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+1. Click the context menu on your home node (***...***) and select the page
+2. Name the page
+3. Add the Widgets you want in your template and set it up how you like it
+
+:::note It is recommended to not fill out the widgets, because if you do, then you will have to delete the content you don't want each time you use the template.:::
+
+### Making the Content Template
+
+Now that we have a page that we want to use for our template.
```
Comment (sofietoft) [inline Umbraco-Uno/Creating-Content/How-To-Setup-Content-Template/index.md:31]:
```suggestion
Now that we have a page that we want to use, follow the steps below to create the Content Template.
```

File: Umbraco-Uno/Creating-Content/How-To-Setup-Content-Template/index.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+## How to set it up
+
+Setting up your template comes in two parts - first you need to make a page that has the widgets you want, and then turn that into a template.
+
+### Setting up the widgets of the page
+
+Start by setting up the page with some widgets.
+You can read more about [Widgets](../../Uno-pedia/Widgets/index.md) in the Uno-pedia section.
+
+1. Click the context menu on your home node (***...***) and select the page
```
Comment (sofietoft) [inline Umbraco-Uno/Creating-Content/How-To-Setup-Content-Template/index.md:23]: I'm not entirely sure I understand what you mean by "context menu" here? 🤔 

```suggestion
1. Click the context menu on your home node (**...**) and select Page
```

File: Umbraco-Uno/Creating-Content/How-To-Setup-Content-Template/index.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+2. Name the page
+3. Add the Widgets you want in your template and set it up how you like it
+
+:::note It is recommended to not fill out the widgets, because if you do, then you will have to delete the content you don't want each time you use the template.:::
+
+### Making the Content Template
+
+Now that we have a page that we want to use for our template.
+
+1. Click the (***...***) button in the tree at the page, you created in the steps above
```
Comment (sofietoft) [inline Umbraco-Uno/Creating-Content/How-To-Setup-Content-Template/index.md:33]:
```suggestion
1. In the content tree, click the three dots on the page you created in the steps above
```

File: Umbraco-Uno/Creating-Content/How-To-Setup-Content-Template/index.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+
+![Three dots icon](images/Three-Dots.png)
+
+2. Select ***Do something else***
```
Comment (sofietoft) [inline Umbraco-Uno/Creating-Content/How-To-Setup-Content-Template/index.md:37]:
```suggestion
2. Select ***Do something else*** - which is at the bottom of the menu
```

File: Umbraco-Uno/Creating-Content/How-To-Setup-Content-Template/index.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+3. On the list choose ***Create Content Template***
+4. Choose the name you want for your template and click ***Create***
+5. Now if you go to create a new page, the template you made earlier in this guide will be there as an option
```
Comment (sofietoft) [inline Umbraco-Uno/Creating-Content/How-To-Setup-Content-Template/index.md:40]:
```suggestion
5. Now if you go to create a new Page, the template you made earlier in this guide will be there as an option
```

**Pull request merged**